apps/nextjs/src/stores/chatStore/index.ts

```diff
@@ -87,10 +99,24 @@ export const useChatStore = create<ChatStore>((set, get) => ({
   setMessages: handleSetMessages(set, get),
 
   // reset
-  reset: ({ ailaStreamingStatus = "Idle", queuedUserAction }) => {
+  reset: () => {
     set({
-      ailaStreamingStatus: ailaStreamingStatus,
-      queuedUserAction: queuedUserAction,
+      id: "",
+      ailaStreamingStatus: "Idle",
+      stableMessages: [],
+      streamingMessage: null,
+      queuedUserAction: null,
+      isExecutingQueuedAction: false,
+      lessonPlan: null,
+      chatAreaRef: { current: null },
+      input: "",
+      isStreaming: false,
+      isLoading: false,
+      actions: {
+        stop: () => {},
+        reload: () => {},
+        append: async () => "",
+      },
```

Comment:
I would think about extracting this to an object, and initialising the store with the same object